.github/workflows/conformance-delegated-ipam.yaml

```diff
@@ -0,0 +1,342 @@
+    steps:
+      - name: Collect Workflow Telemetry
+        uses: catchpoint/workflow-telemetry-action@94c3c3d9567a0205de6da68a76c428ce4e769af1 # v2.0.0
+        with:
+          comment_on_pr: false
+
+      - name: Checkout context ref (trusted)
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+        with:
+          ref: ${{ inputs.context-ref || github.sha }}
+          persist-credentials: false
+
+      - name: Set Environment Variables
+        uses: ./.github/actions/set-env-variables
+
+      - name: Get Cilium's default values
+        id: default_vars
+        uses: ./.github/actions/helm-default
+        with:
+          image-tag: ${{ inputs.SHA }}
+          chart-dir: ./untrusted/install/kubernetes/cilium
+
+      - name: Set up job variables
+        id: vars
+        run: |
+          if [ "${{ github.event_name }}" = "workflow_dispatch" ]; then
+            CONTEXT_REF="${{ inputs.context-ref }}"
+            OWNER="${{ inputs.PR-number }}"
+          else
+            CONTEXT_REF="${{ github.sha }}"
+            OWNER="${{ github.ref_name }}"
+            OWNER="${OWNER//[.\/]/-}"
+          fi
+
+          echo sha=${{ steps.default_vars.outputs.sha }} >> $GITHUB_OUTPUT
+          echo context-ref=${CONTEXT_REF} >> $GITHUB_OUTPUT
+          echo owner=${OWNER} >> $GITHUB_OUTPUT
+
+          # Cilium configured with delegated IPAM mode.
+          # * Set cni.customConf=true since conflist is configured using host mount into kind nodes.
+          # * Delegated IPAM requires direct routing mode.
+          # * Delegated IPAM is incompatible with all options that require cilium-agent to assign itself an IP address,
+          #    so set local-router-ipv4 and endpointHealthChecking.enabled=false.
+          # * Use BPF masquerade with ipMasqAgent.enabled=true because iptables masquerade (enable-ipv4-masquerade=true)
+          #    matches on source IP in the node pod CIDR, which isn't available to Cilium in delegated IPAM mode.
+          CILIUM_INSTALL_DEFAULTS="${{ steps.default_vars.outputs.cilium_install_defaults }} \
+            --helm-set=ipam.mode=delegated-plugin \
+            --helm-set=cni.customConf=true \
+            --helm-set=routingMode=native \
+            --helm-set=ipv4NativeRoutingCIDR=10.244.0.0/16 \
+            --helm-set=endpointRoutes.enabled=true \
+            --helm-set=endpointHealthChecking.enabled=false \
+            --helm-set=extraArgs[0]="--local-router-ipv4=169.254.23.0" \
+            --helm-set=enableIPv4Masquerade=true \
+            --helm-set=bpf.masquerade=true \
+            --helm-set=ipMasqAgent.enabled=true \
+            --helm-set=nodePort.enabled=true"
+
+          CONNECTIVITY_TEST_DEFAULTS="--test-concurrency=5 \
+            --flow-validation=disabled --hubble=false --collect-sysdump-on-failure \
+            --external-target bing.com. --external-cidr 8.0.0.0/8 --external-ip 8.8.4.4 --external-other-ip 8.8.8.8"
```

Comment:
nit: I can potentially see `--external-target bing.com` being a source of flakiness, though we do rely on this for existing AKS testing. I don't see this flag being specified for other kind tests, so it's probably falling back to whatever is the CLI default. Maybe we leave this as-is for now and consider changing it if/when we detect that this is causing issues.

Reply:
oh, I copied this along with the other boilerplate from the conformance-multi-pool test https://github.com/cilium/cilium/blob/d30617430ee8056a979e75b771aea3454b6c18ab/.github/workflows/conformance-multi-pool.yaml#L159

seems like the default value is "one.one.one.one"
https://github.com/cilium/cilium/blob/d30617430ee8056a979e75b771aea3454b6c18ab/cilium-cli/cli/connectivity.go#L129

> Maybe we leave this as-is for now and consider changing it if/when we detect that this is causing issues.

That works for me, I'll leave it for now. Thanks!